tests/atomizer.js

```diff
@@ -307,6 +307,53 @@ describe('getConfig()', function () {
         config = atomizer.getConfig(classNames, defaultConfig);
         expect(config).to.deep.equal(atomizer.mergeConfigs([defaultConfig, expectedConfig]));
     });
+
+    it("should append the default unit", function () {
+        var classNames = ['P-1', 'M-0.3', 'P-2%', 'C-333'];
+
+        var defaultConfig = {
+            defaultUnit: 'px',
+            margin: {
+                defaultUnit: 'em'
```

Comment:
is not about effort only is more about duplication, why do we need to append `px` over and over if we already decided that all paddings should use `px`?

This feature is very similar to the custom breakpoints and as i said it is optional people don't have to use it if they don't want.